guides/Getting-Started.md

```diff
@@ -258,15 +260,41 @@ Spago can be used to turn our PureScript code into JavaScript suitable for use i
     Build succeeded.
     Bundle succeeded and output file to index.js
 
-All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document. If you try this, you should see the words "Hello, World!" printed to your browser's console.
+All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document.  Try this by creating the following `index.html` file in your `my-project` directory:
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Euler Exercise</title>
+</head>
+
+<body>
+  <div id="container"></div>
```

Comment:
I don't think this div is doing anything, so could we remove it?

Reply:
Removed.